base/ryu/exp.jl

```diff
@@ -1,7 +1,7 @@
-function writeexp(buf, pos, v::T,
+@propagate_inbounds function writeexp(buf, pos, v::T,
```

Comment:
I don't think these functions should be inlined.

Reply:
Hmm, I didn't realize that `@propagate_inbounds` automatically inlines it (though that's documented). Maybe I'm out of my depth here, I'll revert the last commit.